great_expectations/data_context/data_context/base_data_context.py

```diff
@@ -128,6 +132,8 @@
     SQLAlchemyError = ge_exceptions.ProfilerError
 
 logger = logging.getLogger(__name__)
+
+# TODO: check if this can be refactored to use YAMLHandler class
 yaml = YAML()
```

Comment:
From discussion: 
- This is global, and module-level, which makes it difficult to mock. Investigation is still needed